wasi/functions.go

```diff
@@ -0,0 +1,137 @@
+package wasi
+
+const (
+	// ModuleSnapshotPreview1 is the module name WASI functions are exported into
+	// See https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md
+	ModuleSnapshotPreview1 = "wasi_snapshot_preview1"
+
+	// FunctionStart is the name of the nullary function a module must export if it is a WASI Command Module.
+	//
+	// Note: When this is exported FunctionInitialize must not be.
+	// See https://github.com/WebAssembly/WASI/blob/snapshot-01/design/application-abi.md#current-unstable-abi
+	FunctionStart = "_start"
+
+	// FunctionInitialize is the name of the nullary function a module must export if it is a WASI Reactor Module.
+	//
+	// Note: When this is exported FunctionStart must not be.
+	// See https://github.com/WebAssembly/WASI/blob/snapshot-01/design/application-abi.md#current-unstable-abi
+	FunctionInitialize = "_initialize"
+
+	// FunctionArgsGet reads command-line argument data.
+	//
+	// See ImportArgsGet
+	// See FunctionArgsSizesGet
+	// See https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md#-args_getargv-pointerpointeru8-argv_buf-pointeru8---errno
+	FunctionArgsGet = "args_get"
+
+	// ImportArgsGet is the WebAssembly 1.0 (MVP) Text format import of FunctionArgsGet
+	ImportArgsGet = `(import "wasi_snapshot_preview1" "args_get"
```

Comment:
how this text format will be used by end user?

Reply:
I can move it internal, but at some point we need to describe what we support, and without this, there's just a link and a function name. I had imagined this describes literally what we test vs the N interpretations of imports out there.